src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import './ButtonPanel.css'
+import Button from "../Button/Button";
+import PropTypes from "prop-types";
+
+class ButtonPanel extends Component {
+    render() {
+        return (
```

Comment:
Let's try to remove duplication here. We can utilize `map` to make rendering the buttons cleaner.